snykTask/src/index.ts

```diff
@@ -140,6 +140,40 @@ async function showDirectoryListing(
   }
 }
 
+async function runSnykCodeTest(
```

Comment:
Suggestion: the actual Code test is being run [here](https://github.com/snyk/snyk-azure-pipelines-task/pull/189/files#diff-7c914af7be18308e0eadff52061e050bfb1a024cfbdb6315772e2273e1b20cc8R221). This method generates a code result when no issues are found. My suggestion would be to rename it something like `generateCodeResultsWithoutIssues()` to be more precise what it is for.